### PR TITLE
Update go to 1.15

### PR DESCRIPTION
**Pull request by carolynvs**
The build on main is failing because 1.13 isn't supported by the latest version of our go tooling.

### CODE REVIEW

**Review by vdice — COMMENTED** [review]
This looks good to me!  Defer to maintainers @silvin-lubecki and @radu-matei for final approval.

**Review by radu-matei — APPROVED** [review]
Thank you, LGTM.

**Pull request merged**
